sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -261,13 +265,21 @@ def build_list(state, arg):
 
 
 def load_attr(state, arg):
+  """Replaces the top of the stack, TOS, with with
```

Comment:
<!--new_thread; commit:b015963f671ceb39f8033014eb6d4e46a4a734bc; resolved:0-->
Drop the duplicate `with`